src/ecs_cluster/ecs_client.py

```diff
@@ -373,11 +373,10 @@ def ssh_to_service(self, cluster_name, service_arn, task_arn,
         else:
             ip_address = ec2_details['PrivateIpAddress']
 
-        key_name = ec2_details['KeyName']
```

Comment:
Do we need to keep this so that devs can still connect to the non-prd environments without having to specify the key?

Reply:
In theory, no, but I suppose it will reduce calls for help. I'll do something a little smarter